src/udiskslinuxdrive.c

```diff
@@ -50,6 +51,71 @@
 #include "udisksdaemonutil.h"
 #include "udiskslinuxdevice.h"
 
+static const gchar *
+_get_blk_path (UDisksDrive *drv, GDBusMethodInvocation *invocation);
+
+static gboolean
+handle_turn_ident_ledon (UDisksDrive *_drive,
+                         GDBusMethodInvocation *invocation);
+
+static gboolean
+handle_turn_ident_ledoff (UDisksDrive *_drive,
+                          GDBusMethodInvocation *invocation);
+
+static gboolean
+handle_turn_fault_ledon (UDisksDrive *_drive,
+                         GDBusMethodInvocation *invocation);
+
+static gboolean
+handle_turn_fault_ledoff (UDisksDrive *_drive,
+                          GDBusMethodInvocation *invocation);
+
+#define _LED_FUN_GEN(func_name, lsm_func) \
+static gboolean \
+func_name (UDisksDrive *_drive, GDBusMethodInvocation *invocation) \
+{ \
+  lsm_error *lsm_err = NULL; \
+  int lsm_rc = 0; \
+  const char *blk_path = NULL; \
```

Comment:
Sorry, the `lsm_xxx` functions return `int` and require `const char *` and these variables does not used for any glib functions.